worker/lib/workers/qemu.ts

```diff
@@ -418,27 +418,29 @@ class Qemu extends EventEmitter implements Leviathan.Worker {
 			this.references.network = undefined;
 		}
 
-		if (this.references.pool != null) {
-			await this.references.pool.stopAsync();
-			this.references.pool = undefined;
-		}
+		if (signal != null) {
+			if (signal == 'SIGTERM' || signal == 'SIGINT') {
```

Comment:
check for SIGTERM does not really make much sense: SIGTERM cannot be handled by the application; the OS kills the process immediately on this signal.

Reply:
How do you mean SIGTERM cannot be handled by the application? I think the signal you are describing is SIGKILL.

Reply:
How do you mean SIGTERM cannot be handled? Afaik SIGTERM is the default way one should use to gracefully kill a process. 
I believe the signal you are describing is SIGKILL.

Reply:
yep, sorry, I confused myself